.changeset/chatty-sheep-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/picker": minor
+---
+
+Addresses issue with vertical alignment of picker text/placeholder by applying inline-flex in template rather than inline-block. Applies flex-shrink to validation icons to prevent icon from resizing when label is long enough to be truncated.
```

Comment:
```suggestion
Addresses issue with vertical alignment of picker text/placeholder by applying inline-flex in template rather than inline-block. Applies flex-shrink to validation icons to prevent icon from resizing when label is long enough to be truncated.
```
The changeset needs updating to reflect the latest changes.

Reply:
Updated to `Applies flex-shrink to validation icons to prevent icon from resizing when label is long enough to be truncated.` to reflect the reduced scope of the changes.